lib/Controller/Schedule.php

```diff
@@ -331,14 +331,23 @@ function eventData()
                 $title = __('%s scheduled on %s', $row->campaign, $displayGroupList);
             }
 
+            if ($row->recurrenceType == 'Minute' || $row->recurrenceType == 'Hour') {
+                $title .= ', Repeats every ' . $row->recurrenceDetail . ' ' . $row->recurrenceType;
```

Comment:
We will need this to be translatable I think, so:

`__(', Repeats every %s %s', $row->recurrenceDetail, $row->recurrenceType)` 

I think?